src/barril/units/_abstractvaluewithquantity.py

```diff
@@ -33,11 +42,11 @@ def __init__(self, category, value=None, unit=None):
                 value = self._GetDefaultValue(quantity.GetCategoryInfo())
 
         else:
-            if category.__class__ != str:
+            if category.__class__ is not str:
```

Comment:
Any reason why this one wasn't changed to `not isinstance(category , str)` ?

Reply:
Hmm no just oversight, fixed, thanks!